.github/workflows/test.yml

```diff
@@ -0,0 +1,24 @@
+name: Test
```

Comment:
FYI we have CI set up: https://teamcity.jetbrains.com/project/KotlinTools_KotlinxKnit?mode=builds

But it only tests pushed branches, and it is not allowed to push an arbitrary unverified code there, so GH-actions still make sense

Reply:
Oh, I totally missed it! Yeah, if you don't mind, I think GH Actions for tests could be useful here. I'll merge it, but let me know if it needs to be rolled back - I don't think it's critical to have